mlflow/langchain/api_request_parallel_processor.py

```diff
@@ -99,10 +104,16 @@ class _ChatRequest(pydantic.BaseModel, extra="forbid"):
 
 class _ChatChoice(pydantic.BaseModel, extra="forbid"):
     index: int
-    message: _ChatMessage
+    message: _ChatMessage = None
     finish_reason: Optional[str] = None
 
 
+class _ChatChoiceDelta(pydantic.BaseModel):
+    index: int
+    finish_reason: Optional[str] = None
+    delta: Optional[_ChatDeltaMessage] = None
```

Comment:
nit: Is `delta` Optional?

Reply:
I didn't observe the case that  it is None.

Reply:
so it's not optional?

Reply:
I think so.